src/defs.ts

```diff
@@ -168,9 +168,9 @@ export interface Response {
   message?: string
 }
 
-export type Relation = 'administrator' | 'editor' | 'viewer' | 'member'
+export type Relation = 'administrator' | 'editor' | 'contributor' | 'member' | 'guest'
 
-export type GroupRelation = 'administrator' | 'editor' | 'viewer'
+export type GroupRelation = 'administrator' | 'editor' | 'contributor' | 'guest'
```

Comment:
since the group and domain relations are same https://github.com/absmach/magistrala/blob/3355332ebdcc8bfd0b35ef8915a3734462f40a3f/docker/spicedb/schema.zed#L25C1-L70C2

we can have just one type `Relation` 
This will affects some other sections of the code that were importing `GroupRelation`